backend/coins/eth/account.go

```diff
@@ -438,6 +445,7 @@ func (account *Account) FatalError() bool {
 // Close implements accounts.Interface.
 func (account *Account) Close() {
 	defer account.initializedLock.Lock()()
```

Comment:
Double lock looks weird.

in btc/account.go, initializedLock is for both `.initialized` and `.closed`. I think the same could make sense here.

Reply:
makes sense, I also changed the name from `accountClosed` to `closed`.
I guess `initializedLock` should maybe be renamed to `stateLock` or something like this, but not in this PR.